src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests.vb

```diff
@@ -702,6 +702,141 @@ ICatchClause (Exception type: System.Exception) (OperationKind.CatchClause) (Syn
             VerifyOperationTreeAndDiagnosticsForTest(Of CatchBlockSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub TestSubSingleLineLambda()
+            Dim source = <![CDATA[
+Imports System
+
+Class Test
+    Sub Method()
+        Dim a As Action = Sub() Console.Write("hello")'BIND:"Sub() Console.Write("hello")"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IAnonymousFunctionExpression (Symbol: Sub ()) (OperationKind.AnonymousFunctionExpression, Type: null) (Syntax: 'Sub() Conso ... te("hello")')
+  IBlockStatement (3 statements) (OperationKind.BlockStatement, IsImplicit) (Syntax: 'Sub() Conso ... te("hello")')
+    IExpressionStatement (OperationKind.ExpressionStatement) (Syntax: 'Console.Write("hello")')
```

Comment:
>IExpressionStatement [](start = 4, length = 20)

Should this node be Implicit?

Reply:
spec says sub expects statement?

(https://docs.microsoft.com/en-us/dotnet/visual-basic/reference/language-specification/expressions)
```
SingleLineLambda
    : LambdaModifier* 'Function' ( OpenParenthesis ParameterList? CloseParenthesis )? Expression
    | 'Sub' ( OpenParenthesis ParameterList? CloseParenthesis )? Statement
    ;
```
so IExpressionStatement is valid there?

Reply:
tagging @AlekseyTs 

also, syntax does have expression statement there.

![image](https://user-images.githubusercontent.com/1333179/31419113-c3751aea-adee-11e7-8c55-9a95d18f3688.png)